Sprint-3/slideshow/slideshow.js

```diff
@@ -1,8 +1,64 @@
-const images = [
-    "./assets/cute-cat-a.png",
-    "./assets/cute-cat-b.jpg",
-    "./assets/cute-cat-c.jpg",
+let autoBackInterval;
+let autoforwardInterval;
+const imagesForSlider = [
+  "./assets/cute-cat-a.png",
+  "./assets/cute-cat-b.jpg",
+  "./assets/cute-cat-c.jpg",
 ];
 
+let seconds = 1;
 
-// Write your code here
\ No newline at end of file
+const prevButton = document.querySelector("#backward-btn"),
+  nextButton = document.querySelector("#forward-btn"),
+  autoForward = document.querySelector("#autoforward-btn"),
+  autoBack = document.querySelector("#auto"),
+  stopButton = document.querySelector("#stop-btn"),
+  image = document.querySelector("#carousel-img"),
+  secondsDelay = document.getElementById("seconds-input");
+
+let curIndex = 1;
+
+function showSlide(index) {
+  imagesForSlider.forEach((src, i) => {
+    if (i === index) {
+      image.src = imagesForSlider[i];
+    }
+  });
+}
+
+function nextImage() {
+  curIndex = (curIndex + 1) % imagesForSlider.length;
+  showSlide(curIndex);
+}
+function prevImage() {
+  curIndex = (curIndex - 1 + imagesForSlider.length) % imagesForSlider.length;
```

Comment:
Can `(curIndex - 1 + imagesForSlider.length)` ever become larger than or equal to `imagesForSlider.length`?

Reply:
No, re result of % can be from [0, lenth-1]